pages/search/index.tsx

```diff
@@ -104,7 +128,16 @@ export default function Search({
         <meta name="twitter:title" content={metadataTitle} key="tw-title" />
         <title key="main-title">{metadataTitle}</title>
       </Head>
+      {/* this span is here to ensure that screen readers are updated
+      when the page has finished loading. in particular, after applying a sort
+      to the results, focus remains on the selector for accessibility reasons.
+      this span ensures that a screen reader will also alert the user that
+      new search results have been delivered. */}
+      <span className={a11yStyles.screenreaderOnly} aria-live="polite">
+        {headerText}
+      </span>
```

Comment:
As discussed offline, let's make this a new ticket and refine the work and expected result.
Right now, when testing this with voiceover, the "Displaying 1-...." text was read twice but now read at all when the sort by select dropdown was updated.